Vote

Amirbapiri opened the pull request.

Amirbapiri committed:
Created finish_vote

Amirbapiri committed:
Created vote_event

mehdy commented on app.py:
```diff
@@ -69,3 +71,23 @@ def delete_event(title):
+
+
+@app.route("/events/<string:title>/vote", methods=["PUT"])
+def vote_event(title):
```
The business logic of voting is to select one of the dates provided. We don't need a voting system for the event itself.

Amirbapiri committed:
Modify the business logic of voting

This commit modifies the business logic of voting. Event objects
modified also in order to implement voting functionality.
